src/NativeBase/Containers/SettingsContainer.tsx

```diff
@@ -87,19 +101,31 @@ const SettingsContainer = () => {
     setColourModeChoice(newColourMode)
   }
 
+  const { authenticated: isLoggedIn, login, logout } = useAuth()
+
+  const handleLogin = () => {
+    navigate('Login', {})
+  }
+
+  const handleLogout = () => {
+    logout()
+  }
+
+  const featureFlags = useFeatureFlags()
+
   return (
     <ScrollView contentContainerStyle={{ flexGrow: 1 }}>
-      <VStack
-        height="100%"
-        justifyContent="space-between"
-        safeAreaY
-        space={4}
-        padding={4}
-      >
-        <VStack space="4">
+      <VStack flex={1} padding={4} space={4}>
+        <Heading textAlign="center" size="lg" marginTop={8} fontFamily="title">
+          SETTINGS
+        </Heading>
+
+        <View style={{ marginTop: 48 }}>
```

Comment:
Style is not need here, just marginTop={'48px'} will suffice :)

Reply:
Thank fixed now too :)